crev-lib/src/util/mod.rs

```diff
@@ -15,14 +16,23 @@ pub const APP_INFO: app_dirs::AppInfo = app_dirs::AppInfo {
     author: "Dawid Ciężarkiewicz",
 };
 
+fn get_git_default_editor() -> Result<String> {
+    let cfg = git2::Config::open_default()?;
+    Ok(cfg.get_string("core.editor")?)
```

Comment:
What if this is string actually `""` or something? Can it happen?

Reply:
Good question.

If I run `git config --global --unset core.editor` then `cfg.get_string("core.editor")` returns an error. I assume that the configuration fields defaults to being "unset" or something reasonable from Git's perspective.

Reply:
:+1: